doc/storageprofile.md

```diff
@@ -36,19 +36,28 @@ status:
     cloneStrategy: snapshot
 ```
 
-Current version supports the following parameters:
+### Parameters
 - `cloneStrategy` - defines the preferred method for performing a CDI clone
 - `claimPropertySets` contains a list of `claimPropertySet`
   - `accessMode` - contains the desired access modes the volume should have
-  - `volumeMode` - defines what type of volume is required by the claim
+  - `volumeMode` - defines what type of volume is required by the claim  
+  These are ordered by preference, so some things to note are:  
+  &emsp;&emsp;Usually Block is preferred over Filesystem for performance reasons (fewer layers)  
+  &emsp;&emsp;RWX over RWO (live migration support).
```

Comment:
these emsp characters look weird.  Can we use standard whitespace?

Reply:
Dropped